formfactory/models.py

```diff
@@ -15,15 +21,25 @@
 clean_methods.auto_discover()
 
 
-FIELD_TYPES = tuple(
-    (field, field) for field in SETTINGS["field-types"]
-    if issubclass(getattr(forms.fields, field), forms.fields.Field)
-)
+def FIELD_TYPES():
+    fields = ()
+    for content_type, field in SETTINGS["field-types"]:
+        module = importlib.import_module(content_type.replace(".%s" % field, ""))
+        if issubclass(getattr(module, field), forms.fields.Field):
+            fields = fields + ((content_type, field),)
+    return fields
 
-WIDGET_TYPES = tuple(
-    (widget, widget) for widget in SETTINGS["widget-types"]
-    if issubclass(getattr(forms.widgets, widget), forms.widgets.Widget)
-)
+FIELD_TYPES = FIELD_TYPES()
```

Comment:
This is just weird. Can't you decorate `def FIELD_TYPES` with `@property`?

Reply:
Yes, that will work. I do sometimes wonder about my missing the obvious.

Reply:
Ah, the error I have here seems familiar. Making use of properties will pass along a property object instead of the expected tuple/list. This in turn causes errors in django that attempts to use the property as a list/tuple object.

Reply:
I think it has something to do with these not being part of an instantiated instance.